dolphinscheduler-e2e/dolphinscheduler-e2e-core/src/main/java/org/apache/dolphinscheduler/e2e/core/DolphinSchedulerExtension.java

```diff
@@ -73,36 +74,27 @@
     private HostAndPort address;
     private String rootPath;
 
+    private DockerImageName imageName;
```

Comment:
This can be a local variable, it doesn't need to be a field

Reply:
I'll fix it.